src/main/configurations/Translate/Configuration_GenereerIdentificatieEmulator.xml

```diff
@@ -43,9 +43,13 @@
 			<IncreaseIntegerPipe 
                 name="IncrementNewIdentificatie"
                 sessionKey="NewIdentificatie">
-                <Forward name="success" path="UpdateIdentificatie" />
+                <Forward name="success" path="CreateNewId" />
             </IncreaseIntegerPipe>
 
+            <XsltPipe name="CreateNewId" styleSheetName="Common/xsl/CreateIdentication.xslt" storeResultInSessionKey="newId">
+                <Param name="zaak-identificatie-template" value="${zaak-identificatie-template}"/>
```

Comment:
Zaak en document both have their own template string in the DeploymentSpecifics. Probably pass the template as session key to the adapter. While calling it from the genereerZaakIdentificatie and genereerDocumentIdentificatie adapters you can pass the correct one.